core/server/master/src/test/java/alluxio/master/metastore/InodeStoreTest.java

```diff
@@ -77,9 +79,12 @@ public static Iterable<Function<InodeLockManager, InodeStore>> parameters() thro
   }
 
   @Rule
-  public ConfigurationRule mConf =
-      new ConfigurationRule(PropertyKey.MASTER_METASTORE_INODE_CACHE_MAX_SIZE,
-          CACHE_SIZE, Configuration.modifiableGlobal());
+  public ConfigurationRule mConf = new ConfigurationRule(
```

Comment:
@tcrain what do you say about enabling the leak check in these UTs (and more UTs)? The UT will just crash on leak spotted, instead of taking place in real environments which takes much more effort to locate and fix (because we can't turn `LEAK_DETECTOR_LEVEL` to `PARANOID`.

Reply:
Yes this sounds like a good idea.